packages/axelar-wasm-std/src/snapshot.rs

```diff
@@ -11,6 +11,22 @@ pub struct Participant {
     pub weight: nonempty::Uint256,
 }
 
+#[cw_serde]
+pub struct NonWeightedParticipant {
```

Comment:
Do we see a future where weighted and non-weighted participants both exist in the system? If no, it makes no sense to me to create this struct.